bladex/profilebase.py

```diff
@@ -202,6 +202,62 @@ def compute_camber_line(self, interpolate=False, n_interpolated_points=500):
             self.camber_line = np.array(
                                [self.xup_coordinates, cl_y_coordinates])
 
+    def deform_camber_line(self, max_camber_change_percent=None,
+                           interpolate=False, n_interpolated_points=500):
+        """
+        Deforms camber line according to a given percentage of change of the
+        maximum camber. Also reconstructs the deformed airfoil's coordinates.
+
+        The percentage of change is defined as follows:
+        .. math::
+            \\frac{new magnitude of max camber - old magnitude of maximum \
+             camber}{old magnitude of maximum camber} * 100
+
+        A positive percentage means the new camber is larger than the max
+        camber value, while a negative percentage indicates the new value is smaller.
+
+        We note that the method works only on airfoils in the reference
+        position, i.e. without any rotations performed, since the measurements
+        are based on the Y-values of the airfoil coordinates, hence any
+        measurements will be inaccurate for rotated foils.
+
+        :param float max_camber_change_percent: percentage of change of the
+            maximum camber. Default value is None
+        :param bool interpolate:  if True, the interpolated coordinates are
+            used to compute the camber line and foil's thickness, otherwise
+            the original discrete coordinates are used. Default value is False.
+        :param n_interpolated_points: number of points to be used for the
+            uniform interpolation, and is used only if parameter interpolate
+            is True. Default value is 500
+        :raises ValueError: if max_camber_change_percent is None
+        """
+        if max_camber_change_percent is None:
+            raise ValueError('max_camber_change_percent must be a valid float')
+
+        # Updating camber line
+        self.compute_camber_line(interpolate=interpolate,
+                                 n_interpolated_points=n_interpolated_points)
+        scaling_factor = max_camber_change_percent / 100. + 1.
+        self.camber_line[1][1:-2] *= scaling_factor     # except LE and TE
```

Comment:
since you are assuming the foil in the reference position you don't need to exclude LE and TE